Libraries/LibWeb/PixelUnits.h

```diff
@@ -405,6 +405,8 @@ constexpr CSSPixels CSSPixels::operator/(CSSPixelFraction const& other) const
 {
     i64 wide_value = raw_value();
     wide_value *= other.denominator().raw_value();
+    if (other.numerator().raw_value() == 0)
```

Comment:
I would rather make caller perform this check. making `operator/` silence division by zero errors will make it harder to find other bugs which are currently discoverable by checking if value is saturated.